containers/Dockerfile.microcosm

```diff
@@ -0,0 +1,34 @@
+FROM golang:1.7
+MAINTAINER <>
+
+RUN apt-get -y update
+RUN apt-get -y install curl iproute2 netbase vim screen openssl
+
+ARG WEB_DOMAIN=dev.microco.sm
```

Comment:
Whilst the API is still a prod one, I'll create a dev site on that for this... probably actually dev.microco.sm .
I'll then use a spare domain name to CNAME to it... dev.microcosmcc.com.

Which means:
* dev.microco.sm = real live API for a dev website
* dev.microcosmcc.com = I'll point this to the docker address

So the docker should get the dev.microcosmcc.com domain.

I'll get this done this evening, will test and then merge.